### PR TITLE
Fixes gh-2420 | Added to test to verify open feign annotations are ignored

### PR DESCRIPTION
**Pull request by thirunar**
This is reference to the issue #2420

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@thirunar Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-netflix&pullRequestId=2435)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-netflix&pullRequestId=2435) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@thirunar Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-netflix&pullRequestId=2435)!

File: spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/feign/valid/FeignClientTests.java [hunk 1]
```diff
@@ -34,9 +34,12 @@
 import java.util.List;
 import java.util.Locale;
 import java.util.Objects;
+import java.util.Map;
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
 
+import com.google.common.collect.ImmutableMap;
```
Comment (spencergibb) [inline spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/feign/valid/FeignClientTests.java]: We do not allow `com.google` classes.

Comment (thirunar) [reply spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/feign/valid/FeignClientTests.java]: I will update to regular Map. The reason for the test is, I was stuck in a situation where I was using HeaderMap in the FeignClient, which converted the Http method from GET to POST, as it was adding it to the body. Had this test been there, I wouldn't have spent time in researching. Let me know if this helps?

**Comment by spencergibb** [comment]
We document that feign annotations don't work be default. If you think the documentation needs clarifying, great. But we don't need a test to verify this.

**Pull request closed without merging**